README.md

```diff
@@ -139,8 +139,8 @@ Key | Description | Default
 --- | --- | ---
 `fieldNameSize` | Max field name size | 100 bytes
 `fieldSize` | Max field value size | 1MB
-`fields` | Max number of non-file fields | Infinity
-`fileSize` | For multipart forms, the max file size (in bytes) | Infinity
+`fields` | Max number of non-file fields | 100 
+`fileSize` | For multipart forms, the max file size (in bytes) | 200MB
```

Comment:
```suggestion
`fileSize` | For multipart forms, the max file size (in bytes) | 209715200 (200MB)
```
Let's be specific here about the default value `209715200 (200mb)`

Reply:
@jonchurch I was being consistent with how the other default values are rendered.  The table has a `Default` column. Do you think that should be changed?

Reply:
Oh nevermind - you're suggesting adding the number of bytes.   that wasn't done for `fieldSize` so I was trying to be consistent

Reply:
Yeah that's what made me think about the human readable strings. When scanning this document I can see someone using `'10mb'` as the value based on the default value (even though it definitely says that the value should be in bytes)